### DISCUSSION TITLE
Configuration example on how to run test locally on single booted iOS simulator

### DISCUSSION THREAD

**Discussion by mzivkovic-symphony** [AppiumTestDistribution/AppiumTestDistribution#1178] [Q&A]
Hello :)

I am pretty much new to ATD and it's ecosystem. I am trying to setup framework locally on my mac machine just to make some proof of concept. I am struggling with configuring local test run on single booted iOS simulator. I was wondering if anyone is willing to provide me with correct capabilities to achieve such a thing. My current capabilities.json file looks like this
`{
  "iOS": {
    "platformName": "iOS",
    "app": "./build/Runner.app",
    "deviceName": "iPhone",
    "preventWDAAttachments": true,
    "clearSystemFiles": true,
    "shouldUseSingletonTestManager": false,
    "simpleIsVisibleCheck": true,
    "maxTypingFrequency": 10,
    "automationName": "XCUITest",
    "usePrebuiltWDA": true
  },
  "serverConfig" : {
    "server": {
      "use-plugins": ["device-farm", "relaxed-caps"],
      "plugin": {
        "device-farm": {
          "platform": "iOS",
          "skipChromeDownload": true,
          "iosDeviceType": "simulated"
        }
      }
    }
  },
  "appiumServerPath": "./node_modules/appium/build/lib/main.js"
}` 

And once i try to run my simple test with command `Platform='ios' mvn clean -Dtest=Runner test` i am getting error
`java.lang.RuntimeException: unable to call device farm endpoints java.io.IOException: HTTP/1.1 header parser received no bytes
        at com.appium.utils.Api.getResponse(Api.java:29)
        at com.appium.device.Devices.getConnectedDevices(Devices.java:26)
        at com.appium.manager.ATDRunner.<init>(ATDRunner.java:41)
        at com.he.lu.mabile.tests.Runner.testRunner(Runner.java:13)
`

For sure i am missing something. Any help would be appreciated a lot.

**Comment by saikrishna321** [comment]
Will take a look

**Reply by saikrishna321** [reply]
@mzivkovic-symphony Please try the latest version of ATD v14.0.4 and the latest RC version appium-device-farm
